src/lib/operators/join_hash/join_hash_steps.hpp

```diff
@@ -191,6 +191,21 @@ class PosHashTable {
     return _offset_hash_table.find(casted_value) != _offset_hash_table.end();
   }
 
+  // Return the number of distinct values (i.e., the size of the hash table).
+  size_t get_distinct_value_count() const {
+    return _offset_hash_table.size();
+  }
+
+  // Return the number of positions stored. For semi/anti joins, no positions are stored and thus 0 is returned.
+  // Otherwise, we use the unified position list that is created in finalize().
+  size_t get_position_count() const {
+    if (_mode == JoinHashBuildMode::AllPositions) {
+      Assert(_unified_pos_list, "");
```

Comment:
Text?

Reply:
`Assert(_unified_pos_list, "Expected unified position list to be set (i.e., hash table is finalized).");`

Reply:
`(i.e., finalize() has been called)`? Just to remove one level of textual indirection.